integration/plugin/common/plugin_test.go

```diff
@@ -276,11 +276,7 @@ func TestPluginBackCompatMediaTypes(t *testing.T) {
 	assert.NilError(t, err)
 	defer rdr.Close()
 
-	type manifest struct {
-		MediaType string
-		v1.Manifest
-	}
-	var m manifest
+	var m v1.Manifest
```

Comment:
Changed this to see if that fixes it (the `v1.Manifest` now has a `MediaType` field again, so I suspect it was masking the field we were checking (this was added as part of https://github.com/moby/moby/pull/42210)